kinto/config/__init__.py

```diff
@@ -1,8 +1,8 @@
 import os
 import codecs
-
```

Comment:
The spaces are actually legitimate here: in python, you:

- first import standard library modules (hint: time is one)
- then import 3rd party library modules
- then import the modules from you project (here kinto).

Between each block you need an empty line.